packages/frontend/src/components/cart/Checkout_test.tsx

```diff
@@ -33,370 +41,440 @@ type CurrencyTestConfig = {
-  {
-    name: "ETH",
-    address: zeroAddress,
-    expectedSymbol: "ETH",
-    isContract: false,
-    expectedTotalAmount: "0.0035 ETH",
-  },
-  {
-    name: "ERC20 (EDD)",
-    address: abi.eddiesAddress as `0x${string}`,
-    expectedSymbol: "EDD",
-    isContract: true,
-    expectedTotalAmount: "5.25 EDD", // EDD has 2 decimals
-  },
-];
-
-async function runCheckoutTest(currencyConfig: CurrencyTestConfig) {
-  const user = userEvent.setup();
-  const shopId = random256BigInt();
-
-  // Set up merchant first
-  const {
-    stateManager: merchantStateManager,
-    relayClient: merchantRelayClient,
-    testAccount: testMerchantAccount,
-  } = await createRouterWrapper({
-    shopId,
-    createShop: true,
-    enrollMerchant: true,
-  });
-  merchantStateManager.addConnection(merchantRelayClient);
-  // wait for manifest to sync
-  await waitFor(async () => {
-    const manifest = await merchantStateManager.get(["Manifest"]);
-    expect(manifest).toBeDefined();
-    const manifestObj = Manifest.fromCBOR(
-      manifest as Map<CodecKey, CodecValue>,
-    );
-    expect(manifestObj.ShopID).toBe(shopId);
-  });
-
-  // Add manifest data with the test currency
-  const testCurrency = new ChainAddress(
-    31337,
-    hexToBytes(currencyConfig.address),
-  );
-
-  // work around it by replacing the whole manifest instead
-  const currentManifestMap = await merchantStateManager.get(["Manifest"]);
-  if (!currentManifestMap) {
-    throw new Error("Manifest not found");
-  }
-  const currentManifest = Manifest.fromCBOR(currentManifestMap);
-  currentManifest.AcceptedCurrencies.addAddress(
-    testCurrency.ChainID,
-    testCurrency.Address,
-    currencyConfig.isContract,
-  );
-  currentManifest.Payees.addAddress(
-    testCurrency.ChainID,
-    fromHex(testMerchantAccount, { size: 20, to: "bytes" }),
-    false,
-  );
-  currentManifest.ShippingRegions = new ShippingRegionsMap(
-    new Map([
-      [
-        "default",
-        new ShippingRegion(""),
-      ],
-    ]),
-  );
-  await merchantStateManager.set(["Manifest"], currentManifest);
-
-  // Create listings
-  for (const [listingID, listing] of allListings.entries()) {
-    // increase price of the test items to make sure we get more then a cent in EDD
-    if (listingID === 23) {
-      listing.Price = 300000000000000n;
-    } else if (listingID === 42) {
-      listing.Price = 700000000000000n;
-    }
-    await merchantStateManager.set(["Listings", listingID], listing);
-    await merchantStateManager.set(["Inventory", listingID], 100);
-  }
+const ETH_CONFIG: CurrencyTestConfig = {
+  name: "ETH",
+  address: zeroAddress,
+  expectedSymbol: "ETH",
+  isContract: false,
+  expectedTotalAmount: "0.0035 ETH",
+};
 
-  // Remove merchant's keycard to free up for customer
-  localStorage.removeItem(`keycard${shopId}`);
+const EDD_CONFIG: CurrencyTestConfig = {
+  name: "ERC20 (EDD)",
+  address: abi.eddiesAddress as `0x${string}`,
+  expectedSymbol: "EDD",
+  isContract: true,
```

Comment:
well.. this _should actually_ be false but i doesn't really matter for the test as-is.

(with `isContract:true`, the payment system assumes the payee is another contract that will process the payment. With `false` the amount is just deposited in the address/account)